spec/controllers/users/webauthn_setup_controller_spec.rb

```diff
@@ -191,6 +192,47 @@
           expect(response).to redirect_to(auth_method_confirmation_url)
         end
       end
+
+      context 'with a single MFA method chosen on account creation' do
```

Comment:
```suggestion
      context 'with only webauthn_platform chosen' do
```

This "with a single MFA method chosen on account creation" context already exists, so I'd use something a litlte more specific.

Reply:
Ah, sounds good - I thought the "it should" difference would be enough to distinguish them, but I'm happy to make that change 👍.